src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -192,29 +192,38 @@ bool MulticopterLandDetector::_get_ground_contact_state()
 	const bool manual_control_idle = (_has_manual_control_present() && _manual.z < _params.manual_stick_down_threshold);
 	const bool manual_control_idle_or_auto = manual_control_idle || !_control_mode.flag_control_manual_enabled;
 
-	// Widen acceptance thresholds for landed state right after landed
-	// so that motor spool-up and other effects do not trigger false negatives.
-	float landThresholdFactor = 1.0f;
-
-	if (hrt_elapsed_time(&_landed_time) < LAND_DETECTOR_LAND_PHASE_TIME_US) {
-		landThresholdFactor = 2.5f;
-	}
+	// land speed threshold
```

Comment:
this comment says nothing, a descripion even if only with keywords would help more then duplicating the title.